### DISCUSSION TITLE
Ligand placement

### DISCUSSION THREAD

**Discussion by RHoejland** [chaidiscovery/chai-lab#84] [Q&A] [answered]
Hi Chai team,
Is there a way to include an initial guess for ligand positioning? I seem to not get correct ligand positioning, when referencing with known crystal structures. Is there a feature, now or on the way, to set a "guess" for the ligand interaction?

**Comment by arogozhnikov** [comment, marked as answer]
Hi Rune, 

not yet, but we will add an example of providing model with a "guess"

Update: guess can now be provided to model as restraint(s), example is here:
https://github.com/chaidiscovery/chai-lab/tree/main/examples/restraints

**Reply by lmolony** [reply]
Hello, where is the example for providing the model with a "guess"?

**Reply by arogozhnikov** [reply]
guess is provided to model as restraint(s), example is here:
https://github.com/chaidiscovery/chai-lab/tree/main/examples/restraints
